### ISSUE TITLE
Plex Media Scanner command error

### ISSUE THREAD

**Issue by ceptonit** [linuxserver/docker-plex#162]
When running Plex Media Scanner inside the container with the command:
`/usr/lib/plexmediaserver/Plex\ Media\ Scanner --help`
I get the following error:
`error while loading shared libraries: libva.so.2: cannot open shared object file: No such file or directory`

Looking at `/usr/lib/plexmediaserver/lib` and "libva.so.2" is right there.
I also checked `/etc/services.d/plex/run` and the file looks correct:
```
#!/usr/bin/with-contenv bash

echo "Starting Plex Media Server."
export PLEX_MEDIA_SERVER_INFO_MODEL=$(uname -m)
export PLEX_MEDIA_SERVER_INFO_PLATFORM_VERSION=$(uname -r)
exec \
	s6-setuidgid abc /bin/bash -c \
	'LD_LIBRARY_PATH=/usr/lib/plexmediaserver:/usr/lib/plexmediaserver/lib /usr/lib/plexmediaserver/Plex\ Media\ Server'
```
Docker log output reports no error:
```
-------------------------------------
          _         ()
         | |  ___   _    __
         | | / __| | |  /  \ 
         | | \__ \ | | | () |
         |_| |___/ |_|  \__/


Brought to you by linuxserver.io
We gratefully accept donations at:
https://www.linuxserver.io/donate/
-------------------------------------
GID/UID
-------------------------------------

User uid:    1000
User gid:    1000
-------------------------------------

[cont-init.d] 10-adduser: exited 0.
[cont-init.d] 40-chown-files: executing... 
[cont-init.d] 40-chown-files: exited 0.
[cont-init.d] 50-gid-video: executing... 
[cont-init.d] 50-gid-video: exited 0.
[cont-init.d] 60-plex-update: executing... 
No update required
[cont-init.d] 60-plex-update: exited 0.
[cont-init.d] done.
[services.d] starting services
Starting Plex Media Server.
[services.d] done.
```

Not sure what's going on.
It used to work but I can't pinpoint after which plex version the scanner stopped working.
I'm guessing maybe after the update to the RUN file?

Docker version: 18.09.3
Plex version: 1.15.1.710

**Comment by aptalca** [comment]
You really should not be manually invoking executables inside the container. But fyi, it's not working because you're not setting the ld library path like the service file is

**Comment by ceptonit** [comment]
Sorry I'm a bit clueless here.
I used to run `docker exec plex /usr/lib/plexmediaserver/Plex\ Media\ Scanner --help` and that would work fine, but now it just returns the "libva.so.2" not found mentioned in OP.
I also just tried a fresh new container and the same command returns the same shared library error but for another lib file.

I guess my question is, is `docker exec plex /usr/lib/plexmediaserver/Plex\ Media\ Scanner --options` not the way to access Plex Media Scanner anymore?
